components/ai_chat/resources/page/chat_ui.tsx

```diff
@@ -98,6 +114,7 @@ function App () {
       onKeyDown={handleSubmit}
       hasSeenAgreement={hasSeenAgreement}
       onHandleAgreeClick={handleAgreeClick}
+      disabled={apiHasError}
```

Comment:
I suppose this is where we can pass through a new boolean state which matches whether a conversation request is in progress

Reply:
added another condition